controllers/review_controller.go

```diff
@@ -26,6 +26,14 @@ func DefaultReviewController(routes *Routes, services *services.Services) *Revie
 }
 
 func (rc *ReviewController) getReview(c *gin.Context) {
+	// is this where I should check if the user's token is valid?
```

Comment:
Since the AuthenticationRequired function is a middleware, it gets called automatically whenever we get a request (to a private endpoint, in our case), so there is no extra work to be done in any of the controller functions.